### ISSUE TITLE
Indicator search doesn't have actual button

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#214]
Kyrgyzstan have picked up that there isn't actually a search button for the search function i.e. it just relies on the user pressing Enter/Return to complete a search. 

Would it be worth adding a button or maybe just to turn the magnifying glass into a button just in case someone doesn't realise they need to press Enter/Return?

**Comment by brockfanning** [comment]
Great point, and it seems like maybe an accessibility issue?

**Comment by brockfanning** [comment]
Fixed by #244
